### ISSUE TITLE
ActivityNotFoundException when trying to open item from ConversationFragment

### ISSUE THREAD

**Issue by otnateos** [intercom/intercom-android#190]
## Version info
  - Intercom for Android version: 1.1.16
  - Android OS version: 4.4.4
  - Android Device: motorola XT1025

## Expected behavior
Based on the stacktrace seems like our user try to open an item in the `ConversationFragment`, so I believe it should have opened the conversation

## Actual behavior
We got ActivityNotFoundException

## Steps to reproduce
Currently we have no way to reproduce. We may need to contact our User about the issue

## Logs

```
android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat=http://www.intercom.io/?utm_source=android-sdk&utm_medium=outbound&utm_campaign=powered-by-intercom flg=0x10000000 }
        at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1723)
        at android.app.Instrumentation.execStartActivity(Instrumentation.java:1608)
        at android.app.Activity.startActivityFromFragment(Activity.java:3974)
        at android.app.Activity.startActivityFromFragment(Activity.java:3949)
        at android.app.Fragment.startActivity(Fragment.java:1054)
        at android.app.Fragment.startActivity(Fragment.java:1033)
        at io.intercom.android.sdk.conversation.ConversationFragment.onItemClick(ConversationFragment.java:250)
        at android.widget.AdapterView.performItemClick(AdapterView.java:299)
        at android.widget.AbsListView.performItemClick(AbsListView.java:1113)
        at android.widget.AbsListView$PerformClick.run(AbsListView.java:2911)
        at android.widget.AbsListView$3.run(AbsListView.java:3645)
        at android.os.Handler.handleCallback(Handler.java:733)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:136)
        at android.app.ActivityThread.main(ActivityThread.java:5086)
        at java.lang.reflect.Method.invokeNative(Method.java:-2)
        at java.lang.reflect.Method.invoke(Method.java:515)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:785)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:601)
        at dalvik.system.NativeStart.main(NativeStart.java:-2)```

**Comment by harkin** [comment]
Thanks for reporting this. 

What happened is a user tapped the "Powered by Intercom" row that appears in conversations. This should open a URL in the browser. I'm testing on a Moto G here and that's the behaviour I'm seeing. 

Do you have any more details about the device, like if it was rooted? How often are you seeing this?

**Comment by harkin** [comment]
I can reproduce this if I disable all browsers on the device

**Comment by harkin** [comment]
I expect it's unusual for someone to disable all their browsers (or somehow end up in that state). 

We can check if the intent can be handled before we fire it off and from the next version onwards we will :+1:

**Comment by otnateos** [comment]
Hi Harkin, thanks for the investigation. 

> What happened is a user tapped the "Powered by Intercom" row that appears in conversations. This should open a URL in the browser. I'm testing on a Moto G here and that's the behaviour I'm seeing.

I can see what you mean when testing on Galaxy S5

> Do you have any more details about the device, like if it was rooted? How often are you seeing this?

We actually received this one report only once, I just thought to investigate straight away since I couldn't find anything from Google search. I believe the user does not have his/her phone rooted. We will keep an eye on the issue and ask the user on how it happened.

**Comment by harkin** [comment]
Investigating early is top :) If you do find out please let me know, I'm very curious!

And if you see any other issues let us know

**Comment by harkin** [comment]
this is fixed now in 1.1.18
